#goal definition
float32  x_start
float32  y_start
float32  depth
bool parallel
float32  ground_position
---
# result
geometry_msgs/Point final     # arm final position
---
# feedback
geometry_msgs/Point current   # arm current position
